src/components/list-item/ListItem.stories.tsx

```diff
@@ -60,36 +38,28 @@ SmallListItem.args = {
   content: "Default",
 };
 
-const ListSpoilerTemplate = ({ showLeftIcon, showRightIcon, content }) => {
+const ListSpoilerTemplate = ({showLeftIcon, showRightIcon, content}) => {
   const [activated, setActivated] = useState(false);
-
+  
   const toggleActivated = () => {
     setActivated((prev) => !prev);
   };
   return (
-    <ListItemDemo onClick={toggleActivated} activated={activated}>
-      <div
-        style={{
-          display: "flex",
-          alignItems: "center",
-          justifyContent: "center",
-          marginRight: 8,
-          transform: `rotate(${activated ? "90deg" : "0"})`,
-          transition: "transform 200ms",
-        }}
-      >
-        <IconChevronRight size={16} />
-      </div>
-      <div style={{ display: "flex", alignItems: "center", justifyContent: "center", gap: 8 }}>
-        {showLeftIcon && <IconStarOutlined size={16} />}
-        <p style={{ width: 200, overflow: "hidden", textOverflow: "ellipsis", whiteSpace: "nowrap" }}>{content}</p>
+    <ListItem onClick={toggleActivated} activated={activated}>
+      <div style={{
```

Comment:
I thought we were about to create this as "Interface" components since this piece of code will be needed in other places.